vscode/microsoft-kiota/src/kiotaInterop/connect.ts

```diff
@@ -24,9 +24,10 @@ export default async function connectToKiota<T>(callback: (connection: rpc.Messa
       console.warn(error);
       const errorMessage = (error as { data?: { message: string } })?.data?.message
           || 'An unknown error occurred';
-    return new Error(errorMessage);
+      return new Error(errorMessage);
   } finally {
       connection.dispose();
+      childProcess.stdin?.end();
```

Comment:
let's close std out as well out of precaution ?

Reply:
on client side I think we only need to close stdin, stdout is only a readable stream, do not have a similar .end() method, killing the process, as is for now, should be enough

Reply:
Thanks for providing more context!